tests/test_examples.py

```diff
@@ -140,8 +140,8 @@ def user_callback(*, value, **kwargs):
     print('time read', time_pv.get())
     print('ctrl read', ctrl_pv.get())
 
-    time_pv.put(3)
-    time_pv.put(6)
+    time_pv.put(3, wait=True)
```

Comment:
You are too kind.  'This worked previously because some yahoo ignored the wait kwarg' :stuck_out_tongue_winking_eye: